app/assets/javascripts/pageflow/ui/views/inputs/text_area_input_view.js

```diff
@@ -22,11 +22,11 @@ pageflow.TextAreaInputView = Backbone.Marionette.ItemView.extend({
       toolbar: this.ui.toolbar[0],
       parserRules: {
         tags: {
-          u: {},
-          b: {},
-          i: {},
+          u: this.options.disableRichtext ? 0 : {},
```

Comment:
Does using `0` here do something special or could it also be `false`? How does the text field behave when inserting text containing formatted text? Is the format removed correctly if `disableRichtext` is `true`?